src/tests/ftest/dfuse/daos_build.py

```diff
@@ -111,22 +113,26 @@ def test_daos_build(self):
             remote_env['D_LOG_FILE'] = '/var/tmp/daos_testing/daos-il.log'
             remote_env['DD_MASK'] = 'all'
             remote_env['DD_SUBSYS'] = 'all'
-            remote_env['D_LOG_MASK'] = 'INFO'
+            remote_env['D_LOG_MASK'] = 'WARN,IL=INFO'
 
         envs = ['export {}={}'.format(env, value) for env, value in remote_env.items()]
 
         preload_cmd = ';'.join(envs)
 
+        build_jobs = 6 * 5
+        if intercept:
+            build_jobs = 1
+
         # Run the deps build in parallel for speed/coverage however the daos build itself does
-        # not yet work, so run this part in serial.
+        # not yet work, so run this part in serial.  The VMs have 6 cores each.
```

Comment:
can we do some auto-detection of core count? this sounds site specific and would have to change if one reconfigures the VM? or runs on HW?